api/tests/prescription/campaign-participation/acceptance/application/pole-emploi-route_test.js

```diff
@@ -119,7 +119,7 @@ describe('Acceptance | API | Pole Emploi envois', function () {
     });
 
     context('When the request has failed', function () {
-      it('should return 403 HTTP status code if user is not allowed to access', async function () {
+      it('should return 200 HTTP status code if user is not allowed to access', async function () {
```

Comment:
Ce test, tel qu'il est décrit est étrange:
`When the request has failed > should return 200 HTTP status code if user is not allowed to access`

Tel qu'il est défini, il semble que ce soit un appel valide, non ?